### ISSUE TITLE
Does not detect if time.monotonic_ns is NotImplemented.

### ISSUE THREAD

**Issue by dglaude** [adafruit/Adafruit_CircuitPython_Debouncer#22]
```
>>> import time
>>> hasattr(time, "monotonic_ns")
True
>>> time.monotonic_ns()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NotImplementedError: No long integer support

```

As a consequence, "my" code fail on the following line and way:
'''
  File "adafruit_debouncer.py", line 112, in update
NotImplementedError: No long integer support
'''

There is a long support discussion with @jepler on Discord.
I might apply the proposed solution for myself, but likely not propose a Pull requests.

**Comment by jepler** [comment]
Hi @dglaude how can we help you so that after you make the fix for yourself you'll be willing to create a pull request and help others benefit as well?

**Comment by dglaude** [comment]
I can share my code if it works. That's no problem. It might just be a few lines. But will it might not be good quality.
Having to use "black" and fight "pylint" and discover other CI check that suddenly hit your code just because last time it was run, the rules were not the same.
That is the kind of things that turn a quick fix into a long fight and deter to contribute.
I know the benefit and value of all of this, but all of those also raise the level.

**Comment by jepler** [comment]
@dglaude I hear you, because it can be a drag—even for seasoned contributors—and since some of the checks are _opinionated_ there are bound to be some each of us will disagree with, but overall we've found it really helps with the consistency and quality of the libraries to do these checks.

It's also important to remember that submitting a PR and having the build be "red" or failed is not a personal failing or mark of shame.  I've found it's easy for me to end up feeling that way anyway, so each time I try to remember that overall I agree with the goals and that this system is helping us deliver better software to our community.

When you're ready, we're available on Discord to help you work through issues with pylint and black.  I do hope and believe that with practice it will feel more routine for you to deal with these messages.
